contrib/ci/minimal-site/site.conf

```diff
@@ -49,6 +49,7 @@
     },
 
     mesh = {
+      vxlan = true,
```

Comment:
This should be not necessary, the default should be "true"?

Reply:
That holds only true in multidomain setups.

@lemoer Wrong `mesh` block :wink: